packages/elastic-safer-lodash-set/scripts/update.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env sh
+
+# Elasticsearch B.V licenses this file to you under the MIT License.
+# See `packages/elastic-safer-lodash-set/LICENSE` more information.
+
+set -e
+
+source ./scripts/_get_lodash.sh
+
+all_files=$(cd lodash && ls)
+modified_lodash_files=(_baseSet.js)
+
+# Get fresh copies of all the files that was originally copied from loadsh,
+# expect the ones in the whitelist
+for file in $all_files
+do
+  if [[ ! "${modified_lodash_files[@]}" =~ "${file}" ]]
+  then
+    cat scripts/license-header.txt > "lodash/$file"
```

Comment:
__question:__ can't we rely on our eslint rule to add this header like it's done for Kibana own files (with `eslint --fix`)?

Reply:
So something like this?

```suggestion
    node scripts/eslint.js "lodash/$file" --fix
```

Reply:
Update: I don't think I can get this to work. If I include the `eslint-disable` comment, it will also affect the missing license header and ESLint will not complain or add it if it's missing. If I remove the `eslint-disable` comment, then ESLint will complain about other stuff in the file. So I think the easiest is to keep it as is, unless you have a workaround?

Reply:
>  If I include the eslint-disable comment, it will also affect the missing license header and ESLint will not complain or add it if it's missing. 

Ahh, I see.

> So I think the easiest is to keep it as is, unless you have a workaround?

Yeah, let's keep it as is. It'd be easier to improve this in a follow up once we merge this PR (I believe it's hard to keep it in sync with constantly changing `master` for too long).